manifests/init.pp

```diff
@@ -120,6 +136,11 @@
     amqp_truststore              => $::certs::candlepin::amqp_truststore,
     qpid_ssl_cert                => $::certs::qpid::client_cert,
     qpid_ssl_key                 => $::certs::qpid::client_key,
+    db_host                      => $katello::candlepin_db_host,
```

Comment:
It is worth mentioning that we need to have the defaults in the class declaration for the installer to see it and display in the help.